addons/website/tests/test_ui.py

```diff
@@ -104,7 +104,8 @@ def test_html_editor_scss(self):
         self.env.ref('base.user_demo').write({
             'groups_id': [(6, 0, [
                 self.env.ref('base.group_user').id,
-                self.env.ref('website.group_website_designer').id
+                self.env.ref('website.group_website_designer').id,
+                self.env.ref('sales_team.group_sale_manager').id,
```

Comment:
As discussed, this should be reverted, business code shouldn't crash on inaccessible records.